availability.php

```diff
@@ -76,6 +76,6 @@
     <form action="availability.php" method="post">
       ' . daily_matrix ( $date, $users ) . '
     </form>
-    ' . print_trailer ( false, true, true );
+    ' . print_trailer ( 0, 1, 1 );
```

Comment:
OK. "Not changing large numbers of files at once" makes more sense. I've been doing global search and individual replace. I do look over the diff file at least six or seven times over a couple of days. And I call up the program in WAMP to look it over. Though I need to upgrade to PHP7 on that. Maybe it would better catch the occasional extra quote, etc.